### PR TITLE
[High Scores & Grade School]:  Synced to Problem Specifications

### PR DESCRIPTION
**Pull request by BethanyG**
- [x] Synced `high scores` problem description.
- [x] Synced `grade school` `.meta/config.json`

### CODE REVIEW

**Comment by BethanyG** [comment]
This updates these exercises from problem specifications, so I think we should get this out soonish.

**Comment by bobahop** [comment]
@BethanyG I will look at this after I return a library book.

**Comment by bobahop** [comment]
High Scores change looks fine. Is `metadata.toml` the source for the Grade School `source_url`? I ask, because `source_url` was deleted about a month ago. Should the new url not be added to that file? The person who deleted the broken link said they could not find a suitable replacement for it, but it appears you did.

**Comment by BethanyG** [comment]
`source_url` was manually added by me to fix `configlet` complaining and failing the CI.
I can PR the URL to problem-specs as well, so that the next time we sync, it's not overwritten.  
Is that the only change here?

**Comment by bobahop** [comment]
Yes, I think it's good to go. I wasn't sure about putting something downstream that isn't upstream, so I'll go ahead and approve now, given the link gets put into prob-specs.

**Review by bobahop — APPROVED** [review]

**Comment by BethanyG** [comment]
Digging more, it looks like there are two gSchools:  [Galvanize](https://www.galvanize.com/full-time), which now points to [Hack Reactor](https://www.galvanize.com/software-engineering-coding-bootcamp-19-weeks) and [gschool](https://gamethinking.io/gschool/).  Judging from others in the [Bolder Area](https://www.builtincolorado.com/2015/02/12/looking-hire-full-stack-developer-here-are-26-gschool-graduates?page=0,0,0,0,0,0,0,0,0,0,1) its **Galvanize**.  

The link on Phil Batto's [LinkedIn profile](https://www.linkedin.com/in/phil-battos/) does indeed lead to a broken link.  It is also a _marketing_ school, rather than a programming school.

Given that this person resides in Bolder, CO - I am going to go with Galvinize.  But it is fairly shady, given that the whole thing is almost ten years old now.  Let me look at the spec and see if that can be a description and not a URL, or if the URL can be omitted.

**Comment by BethanyG** [comment]
Maybe the best thing here is to link to the Person's LinkedIn profile.  I will update accordingly.

File: exercises/practice/grade-school/.meta/config.json [hunk 1]
```diff
@@ -29,5 +29,5 @@
     ]
   },
   "source": "A pairing session with Phil Battos at gSchool",
-  "source_url": "http://gschool.it"
+  "source_url": "https://gamethinking.io/gschool/"
```
Comment (BethanyG) [inline exercises/practice/grade-school/.meta/config.json:32]:
```suggestion
  "source_url": "https://www.linkedin.com/in/phil-battos/"
```

File: exercises/practice/grade-school/.meta/config.json [hunk 2]
```diff
@@ -29,5 +29,5 @@
     ]
   },
   "source": "A pairing session with Phil Battos at gSchool",
-  "source_url": "http://gschool.it"
+  "source_url": "https://www.linkedin.com/in/phil-battos/"
```
Comment (BethanyG) [inline exercises/practice/grade-school/.meta/config.json:32]:
```suggestion
```

File: exercises/practice/grade-school/.meta/config.json [hunk 3]
```diff
@@ -29,5 +29,4 @@
     ]
   },
   "source": "A pairing session with Phil Battos at gSchool",
```
Comment (BethanyG) [inline exercises/practice/grade-school/.meta/config.json:31]:
```suggestion
  "source": "A pairing session with Phil Battos at gSchool"
```

**Pull request merged**